modules/post/windows/gather/credentials/teamviewer_passwords.rb

```diff
@@ -0,0 +1,143 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+#
+# @blurbdust based this code off of https://github.com/rapid7/metasploit-framework/blob/master/modules/post/windows/gather/credentials/gpp.rb
+# and https://github.com/rapid7/metasploit-framework/blob/master/modules/post/windows/gather/enum_ms_product_keys.rb
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Windows::Registry
+
+  def initialize(info={})
+    super(update_info(info,
+        'Name'          => 'Windows Gather TeamViewer Passwords',
+        'Description'   => %q{ This module will find and decrypt stored TeamViewer keys },
+        'License'       => MSF_LICENSE,
+        'References'    => [ ['CVE', '2019-18988'], [ 'URL', 'https://whynotsecurity.com/blog/teamviewer/'] ],
```

Comment:
The CVE here doesn't look valid (https://www.cvedetails.com/cve/CVE-2019-18988/). Is it still reserved?

Reply:
It is still reserved. I requested it to be public on February 3rd when I published the blog post. 
Note the reserve date [here](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2019-18988) matches the timeline in the blog post.